index.html

```diff
@@ -5009,6 +5009,36 @@ <h3>Cryptography Suites and Libraries</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Key Management</h3>
+        <p>
+The security of most digital signature algorithms, which are used to secure
+<a>verifiable credentials</a> and <a>verifiable presentations</a>, is dependent
+on the quality and protection of its <em>private signing key</em>. Guidance in
+the management of cryptographic keys is a large subject and the reader is
+referred to [[NIST-SP-800-57-Part-1]] for more extensive recommendations and
+discussion. As strongly recommended in both [[FIPS-186-5]] and
+[[NIST-SP-800-57-Part-1]], a private signing key is not to be re-used for other
+purposes, such as encryption and signing.
```

Comment:
does this say signing is an "other purpose"?

Reply:
I tried to clarify below... does that change answer the question you're asking, @brianorwhatever ?

Reply:
yes this is more clear now thank you.